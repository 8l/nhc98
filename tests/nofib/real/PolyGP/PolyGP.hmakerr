======	Errors when renaming:
Identifier randomDoubles used at 12:27-12:39 is not defined.
Identifier randomInts used at 12:16-12:25 is not defined.
