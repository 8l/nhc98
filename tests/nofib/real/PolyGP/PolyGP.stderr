Trace On:
create: top fitness is: -30.0
Trace Off.
Trace On:
create: top fitness is: -30.0
Trace Off.
Trace On:
create: top fitness is: -30.0
Trace Off.
Trace On:
create: top fitness is: -30.0
Trace Off.
Trace On:
create: top fitness is: -20.0
Trace Off.
Trace On:
create: top fitness is: -20.0
Trace Off.
Trace On:
create: top fitness is: -20.0
Trace Off.
Trace On:
create: top fitness is: -20.0
Trace Off.
Trace On:
create: top fitness is: -20.0
Trace Off.
Trace On:
create: top fitness is: -20.0
Trace Off.
Trace On:
evolve: top fitness is: -20.0
Trace Off.
Trace On:
evolve: top fitness is: -20.0
Trace Off.
Trace On:
evolve: top fitness is: -20.0
Trace Off.
Trace On:
evolve: top fitness is: -20.0
Trace Off.
Trace On:
evolve: top fitness is: -20.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -8.0
Trace Off.
Trace On:
evolve: top fitness is: -6.0
Trace Off.
Trace On:
The number of evaluation done is the parameter maxEval - 20901"\n"(Lambda "f" (Lambda "l" (Application (Application (Application (Primitive "if-then-else") (Application (Application (Application (Primitive "if-then-else") (Application (Primitive "null") (List []) Boolean) (Arrow Boolean (Arrow Boolean Boolean))) (Application (Primitive "null") (Variable "l") Boolean) (Arrow Boolean Boolean)) (Application (Primitive "head") (List []) Boolean) Boolean) (Arrow (ListType (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2"))))) (Application (Primitive "head") (Application (Application (Application (Primitive "if-then-else") (Application (Primitive "null") (List []) Boolean) (Arrow (ListType (ListType (GenType "G2"))) (Arrow (ListType (ListType (GenType "G2"))) (ListType (ListType (GenType "G2")))))) (Application (Application (Primitive "cons") (List []) (Arrow (ListType (ListType (GenType "G2"))) (ListType (ListType (GenType "G2"))))) (List []) (ListType (ListType (GenType "G2")))) (Arrow (ListType (ListType (GenType "G2"))) (ListType (ListType (GenType "G2"))))) (Application (Primitive "tail") (List []) (ListType (ListType (GenType "G2")))) (ListType (ListType (GenType "G2")))) (ListType (GenType "G2"))) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Primitive "cons") (Application (Primitive "head") (List []) (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Function "map") (Variable "f") (Arrow (ListType (GenType "G1")) (ListType (GenType "G2")))) (Application (Primitive "tail") (Variable "l") (ListType (GenType "G1"))) (ListType (GenType "G2"))) (ListType (GenType "G2"))) (ListType (GenType "G2")))), -30.0)"\n"(Lambda "f" (Lambda "l" (Application (Application (Application (Primitive "if-then-else") (Application (Primitive "null") (Application (Application (Application (Primitive "if-then-else") (Application (Primitive "null") (List []) Boolean) (Arrow (ListType (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2"))))) (Application (Primitive "tail") (List []) (ListType (GenType "G2"))) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Function "map") (Variable "f") (Arrow (ListType (GenType "G1")) (ListType (GenType "G2")))) (List []) (ListType (GenType "G2"))) (ListType (GenType "G2"))) Boolean) (Arrow (ListType (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2"))))) (Application (Application (Primitive "cons") (Application (Variable "f") (Application (Primitive "head") (Variable "l") (GenType "G1")) (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Application (Primitive "if-then-else") (Application (Primitive "null") (List []) Boolean) (Arrow (ListType (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2"))))) (Application (Primitive "tail") (List []) (ListType (GenType "G2"))) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Function "map") (Variable "f") (Arrow (ListType (GenType "G1")) (ListType (GenType "G2")))) (Variable "l") (ListType (GenType "G2"))) (ListType (GenType "G2"))) (ListType (GenType "G2"))) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Application (Primitive "if-then-else") (Application (Primitive "null") (List []) Boolean) (Arrow (ListType (GenType "G2")) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2"))))) (Application (Primitive "head") (Application (Application (Primitive "cons") (List []) (Arrow (ListType (ListType (GenType "G2"))) (ListType (ListType (GenType "G2"))))) (List []) (ListType (ListType (GenType "G2")))) (ListType (GenType "G2"))) (Arrow (ListType (GenType "G2")) (ListType (GenType "G2")))) (Application (Application (Function "map") (Variable "f") (Arrow (ListType (GenType "G1")) (ListType (GenType "G2")))) (List []) (ListType (GenType "G2"))) (ListType (GenType "G2"))) (ListType (GenType "G2")))), -50.0)
Trace Off.
